lib/services/api/api_service.ts

```diff
@@ -16,9 +16,13 @@ const REQUEST_PATH_REGISTER = '/auth';
 const REQUEST_PATH_LOGIN = '/auth/sign_in';
 const REQUEST_PATH_CHANGE_PW = '/auth/change_pw';
 const REQUEST_PATH_SYNC = '/items/sync';
+const REQUEST_PATH_LOGOUT = '/auth/sign_out';
+const REQUEST_PATH_SESSION_REFRESH = '/session/refresh';
 
 const API_VERSION = '20200115';
 
+const EXPIRED_ACCESS_TOKEN_RESPONSE_STATUS = 498;
```

Comment:
I think this should be moved into http_service. 498 is specific to the HTTP protocol and explicitly meant for expired tokens, so it makes sense that http_service would handle this. Then you can expose a function from http_service called `isErrorResponseExpiredToken` which would replace this class' `expiredAccessToken`.